### PR TITLE
Adding support for view.hover hoverSet and leaveSet arguments.

### PR DESCRIPTION
**Pull request by GCheung55**
Addresses #90.

@domoritz Could you give me some advise for testing?

Two things I thought of were:

1. Check the view instance for streams added. I'm not sure where the hover streams are added though.
2. Stub View.hover before embedding to confirm that hover was executed and verify arguments it was executed with.

### CODE REVIEW

File: src/embed.ts [hunk 1]
```diff
@@ -257,17 +257,20 @@ export default async function embed(
     view.tooltip(handler);
   }
 
-  const hover = opt.hover;
+  let hover = opt.hover;
 
   // do not automatically enable hover for Vega-Lite.
```
Comment (domoritz) [inline src/embed.ts]: The code now does "enable hover for Vega if not specified"

Comment (GCheung55) [reply src/embed.ts]: @domoritz Is that incorrect...?

Comment (domoritz) [reply src/embed.ts]: Well, the code doesn't use the negation so the comment shouldn't either.

Comment (GCheung55) [reply src/embed.ts]: I see. I misunderstood that you talking about the code and not the comment.

**Comment by domoritz** [comment]
I'm making an rc 6 release of vl to fix the travis issue.

File: src/embed.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { Config as VlConfig, TopLevelSpec as VlSpec } from 'vega-lite';
 import schemaParser from 'vega-schema-url-parser';
 import * as themes from 'vega-themes';
 import { Handler, Options as TooltipOptions } from 'vega-tooltip';
+import { EncodeEntryName } from 'vega-typings';
```
Comment (domoritz) [inline src/embed.ts]: I just noticed this but let's add this to the Vega lib import above.

Comment (GCheung55) [reply src/embed.ts]: Done!

File: src/embed.ts [hunk 3]
```diff
@@ -28,7 +28,7 @@ export interface Actions {
 
 export interface Hover {
   hoverSet?: EncodeEntryName;
-  leaveSet?: EncodeEntryName;
+  updateSet?: EncodeEntryName;
```
Comment (domoritz) [inline src/embed.ts]: good catch!

File: test/embed.test.ts [hunk 4]
```diff
@@ -113,3 +114,49 @@ test('can change i18n strings', async () => {
...
+    hover: {
+      hoverSet: 'enter',
+    },
+  });
+  expect(hoverSpy).toHaveBeenCalledWith('enter', undefined);
+  hoverSpy.mockReset();
+
+  await embed(el, vgSpec, {
+    hover: {
+      updateSet: 'leave',
```
Comment (domoritz) [inline test/embed.test.ts]: should this be update?

Comment (domoritz) [reply test/embed.test.ts]: or exit

**Comment by domoritz** [comment]
Thank you @GCheung55!

**Pull request merged**